x-pack/plugins/security_solution/cypress/support/index.js

```diff
@@ -21,16 +21,13 @@
 
 // Import commands.js using ES2015 syntax:
 import './commands';
-import 'cypress-promise/register';
```

Comment:
Why was this removed? Can we remove the package as well? It seems unused now.

Reply:
Because we are not using it. I didn't remove the package since now we are using a shared package.json and we are not the only ones using cypress.